src/checked_transaction.rs

```diff
@@ -4,6 +4,7 @@
 //! This allows the VM to accept transactions that have been already verified upstream,
 //! and consolidates logic around fee calculations and free balances.
 
+use crate::consts::BASE_ASSET;
```

Comment:
This is a good idea, although we should probably go one step further and eventually move this into the `ConsensusParameters`. This way it could be configured at runtime for different chains.